cg-spec/editors_draft.html

```diff
@@ -2631,29 +2631,29 @@ <h2>Remarks</h2>
       <section>
         <h2>Implementing the RDF-star semantics</h2>
 
-        <p>It is important to notice that the definitions given <a href="#entailment-of-rdf-star-graphs">above</a> for <a>satisfiability</a> and <a>entailment</a> of <a>RDF-star graphs</a> do not mandate that implementations actually transform <a>RDF-star graphs</a> using the |unstar| mapping. It should be possible to implement the semantics by directly handling RDF-star's <a href="#concepts">abstract syntax</a>.</p>
+        <p>It is important to notice that the definitions given <a href="#entailment-of-rdf-star-graphs">above</a> for <a>satisfiability</a> and <a>entailment</a> of <a>RDF-star graphs</a> do not mandate that implementations actually transform <a>RDF-star graphs</a> using the <a>unstar mapping</a> defined above. It should be possible to implement the semantics by directly handling RDF-star's <a href="#concepts">abstract syntax</a>.</p>
 
       </section>
 
       <section>
         <h2>Combining RDF-star graphs</h2>
 
-        <p>For implementations that rely on the |unstar| mapping defined <a href="#mapping">above</a>, care must be taken when the resulting <a>RDF graphs</a> are combined through union or <a>merge</a>. Given two <a>RDF-star graphs</a> |G| and |H|, it may be the case that |unstar|(|G| ∪ |H|) ≠ |unstar|(|G|) ∪ |unstar|(|H|).</p>
+        <p>For implementations that rely on the <a>unstar mapping</a>, care must be taken when the resulting <a>RDF graphs</a> are combined through union or <a>merge</a>. Given two <a>RDF-star graphs</a> |G| and |H|, it may be the case that <a>unstar</a>(|G| ∪ |H|) ≠ <a>unstar</a>(|G|) ∪ <a>unstar</a>(|H|).</p>
```

Comment:
```suggestion
        <p>Care must be taken when implementations that rely on the <a>unstar mapping</a>, combine the resulting <a>RDF graphs</a> through union or <a>merge</a>. Given two <a>RDF-star graphs</a> |G| and |H|, it may be the case that <a>unstar</a>(|G| ∪ |H|) ≠ <a>unstar</a>(|G|) ∪ <a>unstar</a>(|H|).</p>
```